### PR TITLE
Use proper spacing in susi response files to make it consistent with other files (Fixed #1622)

### LINKED ISSUE #1622: Improper spacing/indentation in susi response files
**Actual Behaviour**

The spacing/indentation is not proper after #1596 

**Expected Behaviour**

Refactor code spacing issues in susi response files.

**Would you like to work on the issue?**

Yes

### PR DESCRIPTION
**Pull request by arundhati24**
Fixes #1622

### CODE REVIEW

File: app/src/main/java/org/fossasia/susi/ai/rest/responses/susi/Datum.java [hunk 1]
```diff
@@ -13,39 +13,51 @@
     @SerializedName("0")
     @Expose
     private String _0;
+
```
Comment (batbrain7) [inline app/src/main/java/org/fossasia/susi/ai/rest/responses/susi/Datum.java]: unnecessary changes

File: app/src/main/java/org/fossasia/susi/ai/rest/responses/susi/Metrics.kt [hunk 2]
```diff
@@ -7,12 +7,18 @@ import com.google.gson.annotations.SerializedName
...
-data class Metrics (
-    val feedback: List<SkillData> = ArrayList(),
-    val usage: List<SkillData> = ArrayList(),
-    val rating: List<SkillData> = ArrayList(),
-    val latest: List<SkillData> = ArrayList(),
-    val newest: List<SkillData> = ArrayList(),
-    @SerializedName("Games, Trivia and Accessories")
-    val topGames: List<SkillData> = ArrayList()
+data class Metrics(
+
```
Comment (batbrain7) [inline app/src/main/java/org/fossasia/susi/ai/rest/responses/susi/Metrics.kt]: why this? everything is clearly visible

File: app/src/main/java/org/fossasia/susi/ai/rest/responses/susi/PostSkillFeedbackResponse.kt [hunk 3]
```diff
@@ -5,8 +5,12 @@ package org.fossasia.susi.ai.rest.responses.susi
  * Created by arundhati24 on 25/06/2018
  */
 data class PostSkillFeedbackResponse(
+
```
Comment (batbrain7) [inline app/src/main/java/org/fossasia/susi/ai/rest/responses/susi/PostSkillFeedbackResponse.kt]: IMHO it is not needed

**Comment by arundhati24** [comment]
@batbrain7 I did this to keep it similar across all files. All changes should be consistent.

**Comment by arundhati24** [comment]
Okay @iamareebjamal 👍 So I will remove the new lines from other files as well to make all changes consistent.

**Comment by arundhati24** [comment]
@iamareebjamal Made the changes. Please have a look.

File: app/src/main/java/org/fossasia/susi/ai/rest/responses/susi/Datum.java [hunk 4]
```diff
@@ -9,7 +9,6 @@
  * <h1>Kotlin Data class to parse data object in retrofit response from susi client.</h1>
  */
 public class Datum extends RealmObject {
-
```
Comment (iamareebjamal) [inline app/src/main/java/org/fossasia/susi/ai/rest/responses/susi/Datum.java]: This is Java class

File: app/src/main/java/org/fossasia/susi/ai/rest/responses/susi/Identity.kt [hunk 5]
```diff
@@ -9,15 +9,12 @@ import java.io.Serializable
  */
 
 class Identity : Serializable {
-
     @SerializedName("name")
     @Expose
```
Comment (iamareebjamal) [inline app/src/main/java/org/fossasia/susi/ai/rest/responses/susi/Identity.kt]: Remove unnecessary expose and serializedname

File: app/src/main/java/org/fossasia/susi/ai/rest/responses/susi/LoginResponse.kt [hunk 6]
```diff
@@ -10,22 +10,17 @@ import com.google.gson.annotations.SerializedName
  */
 
 class LoginResponse {
-
     @SerializedName("message")
     @Expose
```
Comment (iamareebjamal) [inline app/src/main/java/org/fossasia/susi/ai/rest/responses/susi/LoginResponse.kt]: Same

File: app/src/main/java/org/fossasia/susi/ai/rest/responses/susi/LoginResponse.kt [hunk 7]
```diff
@@ -10,22 +10,17 @@ import com.google.gson.annotations.SerializedName
  */
 
 class LoginResponse {
```
Comment (iamareebjamal) [inline app/src/main/java/org/fossasia/susi/ai/rest/responses/susi/LoginResponse.kt]: Make this data class

File: app/src/main/java/org/fossasia/susi/ai/rest/responses/susi/UserSetting.kt [hunk 8]
```diff
@@ -7,21 +7,21 @@ import com.google.gson.annotations.SerializedName
...
-     */
-    @SerializedName("settings")
-    val settings: Settings? = null
+        /**
+         * Gets settings
+         *
+         * @return the settings
+         */
+        @SerializedName("settings")
+        val settings: Settings? = null
```
Comment (iamareebjamal) [inline app/src/main/java/org/fossasia/susi/ai/rest/responses/susi/UserSetting.kt]: Indentation problem

File: app/src/main/java/org/fossasia/susi/ai/rest/responses/susi/LoginResponse.kt (hunk 7 above)
Comment (arundhati24) [reply app/src/main/java/org/fossasia/susi/ai/rest/responses/susi/LoginResponse.kt]: @iamareebjamal  Considering the context of this PR, should I make this change in this PR itself or should I send another PR for this?

File: app/src/main/java/org/fossasia/susi/ai/rest/responses/susi/Datum.java (hunk 4 above)
Comment (arundhati24) [reply app/src/main/java/org/fossasia/susi/ai/rest/responses/susi/Datum.java]: Yes. This too can be converted to data class. Will be sending a PR to check for such classes and convert them to data class. I think one or two classes have still not been converted to data classes yet in susi response package.

File: app/src/main/java/org/fossasia/susi/ai/rest/responses/susi/LoginResponse.kt (hunk 7 above)
Comment (iamareebjamal) [reply app/src/main/java/org/fossasia/susi/ai/rest/responses/susi/LoginResponse.kt]: OK

File: app/src/main/java/org/fossasia/susi/ai/rest/responses/susi/Datum.java (hunk 4 above)
Comment (iamareebjamal) [reply app/src/main/java/org/fossasia/susi/ai/rest/responses/susi/Datum.java]: OK

Comment (arundhati24) [reply app/src/main/java/org/fossasia/susi/ai/rest/responses/susi/Datum.java]: @iamareebjamal Converting this class to a Kotlin data class is leading to issues because there are a number of generated files that inherit this file. And, on compiling I get a number of errors. Should I modify the comment instead?

Comment (iamareebjamal) [reply app/src/main/java/org/fossasia/susi/ai/rest/responses/susi/Datum.java]: OK

File: app/src/main/java/org/fossasia/susi/ai/rest/responses/susi/UserSetting.kt (hunk 8 above)
Comment (arundhati24) [reply app/src/main/java/org/fossasia/susi/ai/rest/responses/susi/UserSetting.kt]: @iamareebjamal I reindented this code, but it remains the same. These spaces appear like this probably because the * is not aligned vertically with @.

Comment (iamareebjamal) [reply app/src/main/java/org/fossasia/susi/ai/rest/responses/susi/UserSetting.kt]: OK
